packages/yourloops/lib/team/hook.tsx

```diff
@@ -83,22 +78,43 @@ function TeamContextImpl(): TeamContext {
     return null
   }
 
-  const getMapUsers = (): Map<string, TeamUser> => {
-    const users = new Map<string, TeamUser>()
-    for (const team of teams) {
-      for (const member of team.members) {
-        if (!users.has(member.user.userid)) {
-          users.set(member.user.userid, member.user)
+  const fetchTeams = useCallback(() => {
+    TeamUtils.loadTeams(user)
```

Comment:
Could you rewrite it with async/await ? (for code consistency)

Reply:
Hummm I thought useEffect did not handle await very well... (not considering IIFE)

Reply:
I won't do it. The cost is too big and annoying for how little the benefit is